Installation.md

```diff
@@ -17,7 +17,8 @@ haxelib install heaps
 ```
 _Note that in some cases you might need to log out and log back into your computer for haxelib to be recognized as a command._
 
-You can also install directly from github to get the latest, bleeding-edge version of Heaps (at the moment this is the recommended option).
+You can also install directly from github to get the latest, bleeding-edge version of Heaps (at the moment this is the **recommended** option).
```

Comment:
Be sure to capture the readers attention what is the recommended way of installing heaps.